src/popover/src/Popover.js

```diff
@@ -47,7 +47,7 @@ const Popover = memo(
         open,
         close
       }),
-      [popoverNode.current]
+      [open, close]
```

Comment:
Looks like the tests are failing because `open` and `close` are used before they are defined. Mind rearranging them to resolve this? @jon-quin

Reply:
done, and manually tested working.